app/src/features/apiClient/screens/apiClient/components/sidebar/components/collectionsList/collectionRow/CollectionRow.scss

```
@@ -22,12 +27,45 @@
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .collection-name {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    font-style: normal;
+    color: var(--requestly-color-text-default);
+    font-size: var(--requestly-font-size-sm, 12px);
+    font-weight: var(--requestly-font-weight-medium);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "font-weight" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)